handlers/chat.go

```diff
@@ -556,6 +557,7 @@ func (ch *ChatHandler) SendMessage(w http.ResponseWriter, r *http.Request) {
 				},
 			},
 		},
+		WebhookURL: fmt.Sprintf("https://community.sphinx.chat/chat/%s/update", request.ChatID),
```

Comment:
This needs to be /hivechat/%s/update

Reply:
@aliraza556 don't create a PR yet. 
There is one more bug assign to Mahtab that needs fixing before we merge any changes here.